.github/workflows/ci-app-e2e-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: CI app E2E tests on main
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+        service_endpoint:
+          description: "The service endpoint URL for the E2E tests"
+          required: true
+          type: string
+        app_name:
+          description: "The application name"
+          required: true
+          type: string
+
+jobs:
+  call-e2e-tests:
+    name: " " # GitHub UI is noisy when calling reusable workflows, so use whitespace for name to reduce noise
+    uses: ./.github/workflows/e2e-tests.yml
+    with:
+      service_endpoint: ${{ inputs.service_endpoint || 'http://app-dev-2093346449.us-east-1.elb.amazonaws.com' }}
```

Comment:
Immediately after merge, can you open a followup PR for adding a CNAME for this ALB @rylew1 ?